pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -293,9 +294,27 @@ func (c *Controller) syncOwnedProvisionRequest(
 				if !psFound || !psaFound {
 					return nil, errInconsistentPodSetAssignments
 				}
+
+				ptName := getProvisioningRequestPodTemplateName(requestName, psName)
+
+				pt := &corev1.PodTemplate{}
+				err := c.client.Get(ctx, types.NamespacedName{Namespace: wl.Namespace, Name: ptName}, pt)
+				if client.IgnoreNotFound(err) != nil {
+					return nil, err
+				}
+				if err != nil {
+					// it's a not found, so create it
+					_, err := c.createPodTemplate(ctx, wl, ptName, ps, psa)
+					if err != nil {
+						msg := fmt.Sprintf("Error creating PodTemplate %q: %v", ptName, err)
```

Comment:
wrap it also in `TruncateEventMessage`, since we are exposing the `err` it might be arbitrarily long (may depend on the user webhook messages)

Reply:
Good catch! Thanks!

Reply:
Done